baselines/dom.generated.d.ts

```diff
@@ -7112,7 +7112,7 @@ interface IDBRequestEventMap {
 }
 
 interface IDBRequest extends EventTarget {
-    readonly error: DOMError;
+    readonly error: DOMException | DOMError;
```

Comment:
Done, but travis seems to have a problem. Please restart the build after it is fixed.